tools/mo/openvino/tools/mo/utils/ir_engine/ir_engine.py

```diff
@@ -67,12 +67,16 @@ def __load_xml(self):
 
         self.graph.name = xml_root.attrib['name'] if xml_root.attrib.get('name') is not None else None
 
+        self.graph.inputs_order = []
```

Comment:
Do we need similar logic for outputs_order? 
Same problem can appear to outputs.

Reply:
Yes, we need.
Implemented.